lib/tests/define.rs

```diff
@@ -703,7 +703,7 @@ async fn define_statement_index_concurrently() -> Result<(), Error> {
 			fields: {},
 			tables: {},
 			indexes: {
-				test: 'DEFINE INDEX test ON user FIELDS email',
+				test: 'DEFINE INDEX test ON user FIELDS email CONCURRENTLY',
```

Comment:
I am wondering if we want options like `CONCURRENTLY`, `OWERWRITE`, `IF EXISTS` to be part of the serialisation on export... Probably interesting to study

Reply:
We do, as they can be part of a `DEFINE FUNCTION` block